server/src/test/java/io/crate/integrationtests/SSLTrustedZoneITest.java

```diff
@@ -70,30 +69,25 @@ protected Settings nodeSettings(int nodeOrdinal) {
             .put("auth.host_based.config.d.protocol", "pg")
             .put(sslSettings);
 
-        switch (nodeOrdinal) {
-            case 0:
-                // This node can switch to plaintext
-                commonSettings.put("auth.host_based.config.a.switch_to_plaintext", "true");
-                break;
-
-            // Node 1 has default value "false" for switch_to_plaintext
+        if (nodeOrdinal == 0) {
+            commonSettings.put("auth.host_based.config.a.switch_to_plaintext", "true");
         }
+        // Node 1 has default value "false" for switch_to_plaintext
         return commonSettings.build();
     }
 
 
-
     @Test
-    @AwaitsFix(bugUrl = "https://github.com/crate/crate/issues/11954")
     public void test_switch_to_plaintext_enabled_downgrades_to_plaintext() throws Exception {
+        internalCluster().startNodes(2);
```

Comment:
```suggestion
        // As soon as the cluster is started, the node will start communicating and downgrading to plaintext.
        // Thus we start the nodes manually in order to just fail this test case and not the suite in case of errors.
        internalCluster().startNodes(2);
```